packages/terra-clinical-result/tests/jest/flowsheet-result-cell/FlowsheetResultCell.test.jsx

```diff
@@ -140,6 +140,28 @@ describe('FlowsheetResultCell', () => {
     expect(cell).toMatchSnapshot();
   });
 
+  it('should render a multiple result stack icon that ignores interpretation due to status', () => {
+    const results = [
+      DefaultResult,
+      {
+        id: '111',
+        systolic: {
+          ...DefaultSystolicResult,
+          status: 'entered-in-error',
+          interpretation: 'critical',
+        },
+        diastolic: {
+          ...DefaultDiastolicResult,
+          status: 'entered-in-error',
+          interpretation: 'critical',
+        },
+      },
+      DefaultBloodPressureResult,
+    ];
+    const cell = shallowWithIntl(<FlowsheetResultCell resultDataSet={results} hideUnit />).dive();
```

Comment:
Is this the same test as above? Is there a way to validate the multiple result stack icon without interpretation?

Reply:
So this test is specifically testing one of the issues we were dealing with, where the multiple result stack icon would get its interpretation color based on an `entered in error` value which shouldn't happen. This test should render a default result in the flow sheet cell and with a multiple icon set that doesn't get the class to render it as `critical` since it is `entered-in-error`

The test above is a test to make sure the both the systolic value and diastolic value render the `enterered-in-error` changes.

Reply:
Should we add a simple assertion check to validate the `critical` classname has not been applied on the icon?

Reply:
Updated the test and test name, https://github.com/cerner/terra-clinical/pull/639/commits/111cb724c4d4c4db05e06c6aa19c7006cf69ccbc